app/views/layouts/application.html.erb

```diff
@@ -1,18 +1,15 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <title>Kata07RelojChecadorK7F22T4</title>
-    <meta name="viewport" content="width=device-width,initial-scale=1">
-    <%= csrf_meta_tags %>
-    <%= csp_meta_tag %>
-
-    <%= stylesheet_link_tag "sass/views/#{params[:controller]}/#{params[:action]}", "data-turbo-track": "reload" %>
-    <%= javascript_importmap_tags %>
-  </head>
-
-  <body>
-    <div class="container">
-      <%= yield %>
-    </div>
-  </body>
+<head>
+  <title>Kata07RelojChecadorK7F22T4</title>
+  <meta name="viewport" content="width=device-width,initial-scale=1">
+  <%= csrf_meta_tags %>
+  <%= csp_meta_tag %>
+  <%= stylesheet_link_tag "application", "navbar", "data-turbo-track": "reload" %>
+  <%= javascript_importmap_tags %>
+</head>
+<body>
+ <%= render "layouts/navbar" %>
+ <%= yield %>
+</body>
```

Comment:
This block need to have an extra space of indentation

Reply:
@JuanequeX

Reply:
Change made